pages/linux/nemo.md

```diff
@@ -1,16 +1,24 @@
 # nemo
 
-> File manager and graphical shell for Cinnamon.
-> More information: <https://github.com/linuxmint/nemo>.
+> Manages files and directories in Cinnamon desktop environment.
+> More information: <https://manned.org/nemo.1>.
 
-- Open a new window showing the user's home directory:
+- Open a current user home directory:
```

Comment:
```suggestion
- Open the current user's home directory:
```

Reply:
I've used `Open the current user home directory:`.